src/vl-pattern.js

```diff
@@ -1,13 +1,13 @@
 import {awaitUntil} from '/node_modules/vl-ui-core/dist/vl-core.js';
 import '/node_modules/@govflanders/vl-ui-util/dist/js/util.js';
 import '/node_modules/@govflanders/vl-ui-core/dist/js/core.js';
-import '/node_modules/@govflanders/vl-ui-pattern/dist/js/pattern.js';
+import '../lib/pattern.js';
```

Comment:
```suggestion
import '/lib/pattern.js';
```
Absoluut path gebruiken om correct gebuild te krijgen met de dist build setup.